.github/ISSUE_TEMPLATE/special_case_issue.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Special Case Issue
+about: A complex or exceptional issue that doesn't fit the usual templates
+
+---
+
+<!-- 
+Please read the text in this edit field before filling it in. 
+If your issue doesn't neatly fit into either the bug report or feature request categories, and it involves a more complex or unique scenario, use this template. 
+Before proceeding, please review the existing templates to ensure that those do not better suit your needs:
+- Bug Report Template: https://github.com/nvaccess/nvda/issues/new?template=bug_report.md
+- Feature Request Template: https://github.com/nvaccess/nvda/issues/new?template=feature_request.md
+
+If required information is not provided when requested, the issue may be closed.
+For guidance on how to fill in this template, including how to provide the required files, please see:
+https://github.com/nvaccess/nvda/blob/master/projectDocs/issues/githubIssueTemplateExplanationAndExamples.md
+
+Please also note that the NVDA project has a Citizen and Contributor Code of Conduct which can be found at https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md. 
+NV Access expects that all contributors and other community members read and abide by the rules set out in this document while participating or contributing to this project. 
+This includes creating or commenting on issues and pull requests.
+
+Questions below start with multiple hash symbols (#). 
+Please place your answers and information on the blank line below each question.
+-->
+
+### Detailed description of the issue
+<!--
+Provide a detailed description of the issue you are reporting. 
+Include specific details to help us understand the context.
+-->
```

Comment:
I would avoid the use of inline comments here in favour of putting the information in the template explainer. It adds a lot of noise to those who regularly fill out issues and often causes formatting problems where users break or put information in the comments themselves.